pkg/v1/tkg/client/delete_region.go

```diff
@@ -470,15 +470,11 @@ func (c *TkgClient) waitForAVIResourceCleanup(regionalClusterClient clusterclien
 		if apierrors.IsNotFound(err) {
 			return nil
 		}
-		return errors.Wrapf(err, "unable to get ako statefulset")
 	}
 	if err := regionalClusterClient.WaitForAVIResourceCleanUp(constants.AkoStatefulSetName, constants.AkoNamespace); err != nil {
-		// losing connection should be consider as AVI resources clean up finished.
-		// TODO: (xudongl) Any ideas how to make this line cleaner? Thanks.
-		if strings.Contains(err.Error(), "dial tcp") {
-			return nil
+		if !strings.Contains(err.Error(), "dial tcp") {
+			log.Error(err, "clean up AVI resources error")
```

Comment:
This will cause cluster-deletion to continue even though AKO is stuck in resource deletion.

Did we decide to use IsServiceUnavailable() instead?